### ISSUE TITLE
Consolidated Card Pages

### ISSUE THREAD

**Issue by ghost** [Null-Signal-Games/netrunnerdb#278]
While I appreciate the consolidation of card pages in general I found some issues with that too.

It is hard to find, where this card is included! E.g. Morningstar is now only part of Revised Core Set. While this is true it is not only part of it, but is also included in Genesis and I can imagine this leading to issues. e.g. when you don't have a revised core set.
Or in my case I was searching for cards explicitly from old core + first cycles.

I have to admit, I hate it, when cards show up multiple times when searching for them. But it would be good when at least the card page would contain where it was released.

It is also difficult when it's not part of a search result.
When I search for cards from "What lies ahead" it is listed as card 4.
When I search for Anarch Fractors it is only listed in revised core set.

I have no final suggestion to solve this completly, but I think updating the card page with all packs where it is contained should be done!

**Comment by ghost** [comment]
ok, I realized there is a small new symbol in the bottom of the description that contains all published versions!
Still think searching can be an issues, but no idea how to solve it.

**Comment by NoahTheDuke** [comment]
Yeah it’s definitely a difficult problem and I’m very willing to hear any solutions you have. This was my best effort but I am not the best designer so please let me know where this is hard to use!
